packages/flutter/test/cupertino/search_field_test.dart

```diff
@@ -111,7 +111,24 @@ void main() {
       );
     },
   );
-
+  testWidgets(
+    'can change keyboard type',
+    (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('can change keyboard type', (WidgetTester tester) async {
```

Those three lines fit in one line